src/ontology/ro.Makefile

```diff
@@ -68,20 +68,6 @@ tmp/missing-base.owl: $(EDIT_PREPROCESSED) $(ONTOLOGYTERMS)
 		filter -T $(ONTOLOGYTERMS) --trim false --axioms "AsymmetricObjectProperty FunctionalObjectProperty FunctionalDataProperty InverseFunctionalObjectProperty SymmetricObjectProperty TransitiveObjectProperty InverseObjectProperties ObjectPropertyDomain DataPropertyDomain DataPropertyRange ObjectPropertyRange" \
  -o $@.tmp.owl && mv $@.tmp.owl $@
 
-# TODO: This goal (and tmp/missing-base.owl) can be removed once https://github.com/ontodev/robot/issues/1108 is resolved
```

Comment:
You also need to remove the `tmp/missing-base.owl` rule, which is above this one, right?

Reply:
Thanks!